### PR TITLE
Proposals & User tab - activity section new layout + filtering

### LINKED ISSUE #807: New User page design update
Update user page to the new design linked below:

Proposals:
https://www.figma.com/file/gXLZPoh1BchigOB5y9YkQZ/Tezos-Homebase---v2-UI?type=design&node-id=8620-5780&mode=design&t=dVP5XuCZoBJL3r7q-4

Votes:
https://www.figma.com/file/gXLZPoh1BchigOB5y9YkQZ/Tezos-Homebase---v2-UI?type=design&node-id=8695-4435&mode=design&t=dVP5XuCZoBJL3r7q-4

Transactions:
https://www.figma.com/file/gXLZPoh1BchigOB5y9YkQZ/Tezos-Homebase---v2-UI?type=design&node-id=8695-4953&mode=design&t=dVP5XuCZoBJL3r7q-4

My Activity (full view of activity):
https://www.figma.com/file/gXLZPoh1BchigOB5y9YkQZ/Tezos-Homebase---v2-UI?type=design&node-id=8693-3836&mode=design&t=dVP5XuCZoBJL3r7q-4

### PR DESCRIPTION
**Pull request by fabiolalombardim**
Closes #807

### CODE REVIEW

**Comment by fabiolalombardim** [comment]
Closes #782

File: src/modules/explorer/pages/Proposals/index.tsx [hunk 1]
```diff
@@ -233,6 +253,18 @@ export const Proposals: React.FC = () => {
...
+          <FiltersContainer
+            onClick={() => setOpenFiltersDialog(true)}
+            xs={isMobileSmall ? 12 : 2}
+            item
+            container
+            direction="row"
+            alignItems="center"
+          >
+            <FilterAltIcon style={{ color: theme.palette.secondary.main }} fontSize="small" />
+            <Typography color="secondary">Filter & Sort</Typography>
```
Comment (Man-Jain) [inline src/modules/explorer/pages/Proposals/index.tsx:265]: Not working for me the first time I select `Active` or `Closed` it just shows No Proposal found. Then second time it works

**Review by Man-Jain — APPROVED** [review]

File: src/modules/explorer/pages/Proposals/index.tsx (hunk 1 above)
Comment (Man-Jain) [reply src/modules/explorer/pages/Proposals/index.tsx:265]: Yes, All good now.

**Comment by Man-Jain** [comment]
LGTM

**Pull request merged**